src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/softmax/softmax_items_class_kernel_base.cpp

```diff
@@ -36,6 +36,7 @@ ParamsKey SoftmaxItemsClassKernelBase::GetDefaultSupportedKey() {
     k.EnableSoftmaxDim(SoftmaxDim::Y);
     k.EnableSoftmaxDim(SoftmaxDim::Z);
     k.EnableSoftmaxDim(SoftmaxDim::FEATURE);
+    k.EnableSoftmaxDim(SoftmaxDim::BATCH);
```

Comment:
Could you please add bsv32_fsv16 layout for input/output? It is required for dGPU.

Reply:
While I understand that those layouts are needed for dGPU support, I think that adding those layouts -- and especially crafting test data for those -- should be done out of the scope of this PR.

Shall we consider a follow-up PR for this task?

Reply:
We are adding supports for blocked formats for all newly added primitives. The test data can be generated in a random way, and the expected data can be derived from the reference code. Please refer to this WIP PR. https://github.com/openvinotoolkit/openvino/pull/11533/files  No need to make same way, (the PR is WIP so we are may refactor more) But I think you can find how to add the blocked formats tcs there and compare with reference code.

Reply:
As was agreed in the meeting with @tgubanova-lohika, adding new layouts are outside of the scope, and should be added outside of this PR.